katsu_ingest.py

```diff
@@ -98,7 +137,7 @@ def ingest_data(katsu_server_url, data_location):
     )
     ingest_finished = False
     for api_name, file_name in file_mapping.items():
-        ingest_str = f"/v2/ingest/{api_name}"
+        ingest_str = f"/katsu/v2/ingest/{api_name}/"
```

Comment:
Same here, I need to remove trailing slash to make it work

Reply:
Added option to set whether Katsu uses trailing slash in [ca4009c](https://github.com/CanDIG/candigv2-ingest/pull/18/commits/ca4009c81f7a566d3281fbc9c5191447223adf37)